Source/UI/NewUI/BillingForm/ViewController/BillingFormViewController.swift

```diff
@@ -17,7 +17,8 @@ protocol BillingFormViewControllerDelegate: AnyObject {
     func doneButtonIsPressed(sender: UIViewController)
     func cancelButtonIsPressed(sender: UIViewController)
     func getViewForHeader(sender: UIViewController) -> UIView?
-    func update(country: Country)
+    func update(country: Country?)
```

Comment:
Does it make sense to update with a nil country? 

Also, 
`update(country: Country?)` 
and
`phoneNumberIsUpdated(number: String?)` 
would appear to do similar things, observe property changes, but their naming is very different. Should they be more similar or do we want them to be as they are now?